### ISSUE TITLE
Menu button text does not match accessible name

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/chapter#2042]
### Describe the bug:

The menu button in the upper right corner has the visible text "Menu" but the accessible name on the button is set to "Options" using `aria-label`. This is going to make it hard for people using voice control software to use the menu since they won't be able to see the actual accessible name. It will also be confusing for screen reader users who can see the text on the page (not all screen reader users are blind).

### Solution

Decide whether this button should be called "Menu" or "Options" and then use that as the button text and get rid of the `aria-label` completely as it is not needed.

**Comment by allella** [comment]
Thanks for your notes.

I'd have to imagine "Options" was a mistake as it's clearly meant to be the main menu.
